homeassistant/components/rfxtrx/cover.py

```diff
@@ -65,13 +66,13 @@ def __init__(
         device: rfxtrxmod.RFXtrxDevice,
         device_id: DeviceTuple,
         event: rfxtrxmod.RFXtrxEvent = None,
-        venetian_blind_mode: bool | None = None,
+        venetian_blind_mode: str | None = None,
```

Comment:
This is used to compare with `CONST_VENETIAN_BLIND_MODE_US` and `CONST_VENETIAN_BLIND_MODE_EU`, which are strings, so I think the type hint was wrong here.